### PR TITLE
Update for Georgia, Ukraine, Lebanon. frequency-plans.mdx

### PR DESCRIPTION
**Pull request by Topius**
Hi 
There was mistake in blocking Georgia, Ukraine, Lebanon from buying Bobcat miners as the Frequency information was not known about Country and was getting: 

Restricted Access

Sorry, you cannot visit our store from your current location.

We don't support the frequency of your country.

Here is an instruction of Helium frequnecy.

https://docs.helium.com/lorawan-on-helium/frequency-plans/ ,  - OLD and Outdated

----------------------------------------------------------------------------------------------------
NEW
Link from Lora-Alliance : https://lora-alliance.org/wp-content/uploads/2020/11/RP_2-1.0.2.pdf
Georgia (GE)
433.05 - 434.79 MHz EU433
863 - 873 MHz EU863-870

Ukraine (UA)
433.05 - 434.79 MHz EU433
863 - 870 MHz EU863-870

Lebanon (LB)
433.05 – 434.79 MHz EU433
863 - 870 MHz EU863-870

I updated information also linked Screenshots on discord was guided by admin / mod to update here. 
Also provided Link to communication agency of Georgian and Frequency spectrum Chart.

And Copy text about Frequency from Georgia's Spectrum chart:
არა-სპეციფიური SRD გამოყენებები (863.0-876.0 მჰც),  >>>Translate >>>> Non-specific SRD applications (863.0-876.0 MHz),

### CODE REVIEW

**Review by Ottobcn — DISMISSED** [review]

**Comment by Topius** [comment]
To check the Legitimacy of my Pull Request here is Link of Communications Commission http://mca.comcom.ge/spectrum_map That I also added to a Chart in Pull  >>>> In there Right Click Translate to English on Chrome >>>>>> Pick range Range 402 MHz - 4.2 GHz >>>>>>> Right button corner of that Range is Button >>>> SHOW TABLE >>>> Scroll down to 862 - 890 MHz and on the right its written >> Non-specific SRD applications (863.0-876.0 MHz),   End Of Frequency is licensed to a company Silknet (872.955-876.645 MHz)   so That makes Rest 863-870 Free for Everyone.

File: docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx [hunk 1]
```diff
@@ -206,7 +206,7 @@ Also known as **US902-928**
...
-| Georgia       | unknown, limited CEPT | CEPT Rec. 70-03                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Georgia       | EU863-870             | CEPT Rec. 70-03    http://mca.comcom.ge/spectrum_map/                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (King-of-Spades) [inline docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx]: I can't independently confirm this based on your instructions. The page you linked doesn't load. 

I tried contacting the commission here: https://comcom.ge/en/contact

They use the same link on their site, so if nothing else maybe they need to fix it. :D

Comment (King-of-Spades) [reply docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx]: Update, I've potentially confirmed the finding based on this resource: https://lora-alliance.org/wp-content/uploads/2020/11/RP_2-1.0.2.pdf

While checking PR (Lebanon): https://github.com/helium/docs/pull/386

Though if accurate, it should be add **EU433**:
```suggestion
| Georgia       | EU863-870   EU433  | [RP002-1.0.2 LoRaWAN® Regional Parameters (Nov. 2020)](https://lora-alliance.org/wp-content/uploads/2020/11/RP_2-1.0.2.pdf)                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment (Topius) [reply docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx]: Thank you for your assistance and that Link from Lora Alliance, Time to talk with admins :)

**Comment by King-of-Spades** [comment]
Should be resolved by https://github.com/helium/docs/pull/418

**Pull request closed without merging**